website/client/src/libs/highlightUsers.js

```diff
@@ -14,9 +14,12 @@ export function highlightUsers (text, userName, displayName) { // eslint-disable
       return fullMatched;
     }
 
+    let fixedStr = mentionStr.replace(/<\/?em>/g, '_');
+    fixedStr = fixedStr.replace(/<\/?strong>/g, '__');
+
     const isUserMention = currentUser.includes(mentionStr) ? 'at-highlight' : '';
```

Comment:
This check should also compare to fixedStr, otherwise you're comparing the name including html tags against the normal user name.

```suggestion
    const isUserMention = currentUser.includes(fixedStr) ? 'at-highlight' : '';
```